Changelog.md

```diff
@@ -1,3 +1,9 @@
+## 2.5.0 - unreleased
+
+### Added
+
+-   Pre-bundled Modem Trace Databases: v2.0.3, and v2.0.3-FOTA.
```

Comment:
Is this the modem trace firmware mentioned in https://docs.nordicsemi.com/bundle/nrf-connect-cellularmonitor/page/requirements.html#software-requirements or some other new package that needs to be mentioned in the docs?

Reply:
yes, comes from here:
https://www.nordicsemi.com/Products/nRF9151/Download?lang=en#infotabs